modules/stochastic_tools/src/samplers/AdaptiveImportanceSamplerActiveLearning.C

```diff
@@ -0,0 +1,28 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "AdaptiveImportanceSamplerActiveLearning.h"
+
+registerMooseObjectAliased("StochasticToolsApp",
+                           AdaptiveImportanceSamplerActiveLearning,
+                           "AISActiveLearning");
+
+InputParameters
+AdaptiveImportanceSamplerActiveLearning::validParams()
+{
+  InputParameters params = AdaptiveImportanceSampler::validParams();
+  params.addClassDescription("Adaptive Importance Sampler with Gaussian Process Active Learning.");
+  return params;
+}
+
+AdaptiveImportanceSamplerActiveLearning::AdaptiveImportanceSamplerActiveLearning(
+    const InputParameters & parameters)
+  : AdaptiveImportanceSampler(parameters)
+{
+}
```

Comment:
I am convinced that we don't need this class if the behavior is already captured in `AdaptiveImportanceSampler`.

Reply:
Please see my response to the previous comment. We can discuss more.